src/framework/theme/components/datepicker/datepicker.component.ts

```diff
@@ -466,6 +468,9 @@ export class NbBasePickerComponent<D, T, P> extends NbBasePicker<D, T, P>
    * */
   @Input() overlayOffset = 8;
 
+  @Input() adjustment: NbAdjustment = NbAdjustment.COUNTERCLOCKWISE;
+  static ngAcceptInputType_adjustment: NbAdjustmentValues;
```

Comment:
I think we don't need `static ngAcceptInputType_adjustment: NbAdjustmentValues;`. We use the `static ngAcceptInputType_` thing only when a property type differs from what the user can enter. Mostly we use it for the boolean inputs which could be used without a value, e.g. `<my-component booleanInput>`. Here the value of `booleanInput` is `undefined`, but the type on the property says it should be boolean, and Angular treat it like an error. With the `static ngAcceptInputType_booleanInput` we could overwrite type from the property with `boolean | undefined` to support usage without a value.